src/test/resources/application-test-C100-form.yaml

```diff
@@ -5,6 +5,16 @@ uk:
         caseTypeofApplication:
           C100:
             mandatoryFields:
+              - previous_or_ongoingProceeding
+              - existingCase_onEmergencyProtection_Care_or_supervisioNorder
+              - exemption_to_attend_MIAM
+              - familyMember_Intimation_on_No_MIAM
+              - attended_MIAM
+              - NoMIAM_domesticViolence
+              - NoMIAM_childProtectionConcerns
+              - NoMIAM_Urgency
+              - NoMIAM_PreviousAttendance
+              - NoMIAM_otherReasons
```

Comment:
need to move optional fileds

Reply:
Done. All conditional mandatory fields now moved to Option Field group